projects/com.oracle.truffle.llvm.tools/src/com/oracle/truffle/llvm/tools/GCC.java

```diff
@@ -62,10 +63,15 @@ public static void compileToLLVMIR(File toBeCompiled, File destinationFile) {
         }
 
         String destinationLLFileName = destinationFile.getAbsolutePath();
-        String interimLLFileName = "/tmp/interim.ll";
+        File interimFile;
+        try {
+            interimFile = File.createTempFile("interim", ".ll");
+        } catch (IOException e) {
+            throw new AssertionError(e);
+        }
 
         if (destinationFile.getName().endsWith(".bc")) {
-            destinationLLFileName = interimLLFileName;
+            destinationLLFileName = interimFile.getName();
```

Comment:
Maybe 'interimFile.getAbsolutePath()' shall do the trick and also it will create files in temp dir as we are using this name in .ll file generation command.